docs/src/ToricVarieties/AlgebraicCycles.md

```diff
@@ -7,9 +7,68 @@ Pages = ["AlgebraicCycles.md"]
 ```
 
 
-# Algebraic Cycles
+# The Chow ring and algebraic Cycles
 
-## Special attributes of toric varieties
+Algebraic cycles are elements of the Chow ring.
+We provide methods to work with such cycles.
+
+## Constructors
+
+### Generic constructors
+
+```@docs
+AlgebraicCycle(d::ToricDivisor)
+AlgebraicCycle(c::ToricDivisorClass)
+AlgebraicCycle(l::ToricLineBundle)
+AlgebraicCycle(cc::CohomologyClass)
+```
+
+### Addition, subtraction and scalar multiplication
+
+Addition of algebraic cycles `ac1` and `ac2` is implemented by
+`ac1+ac2`. Similarly, we can subtract the cycles by `ac1-ac2`.
+Scalar multiplication with `c` (this could be an integer,
+fmpz or even fmpq number) is supported by `c*ac1`.
+
+### Intersection product
+
+The intersection product of two algebraic cycles `ac1` and `ac2`
+is computed by `ac1*ac2`. This makes sense, since algebraic
+cycles on toric varieties are elements of the Chow ring, which
+in turn is (a certain) quotient of the Cox ring. Hence, internally,
+an algebraic cycle can be simply thought of as a polynomial in this
+ring and the intersection product corresponds to the product of two
+(equivalence classes of) polynomials.
+
+We support `ac1^n`, which corresponds to computing the intersection
+product of `ac1` with itself `n`-times.
+
+
+## Properties
+
+To check if an algebraic cycle `ac` is trivial, one can invoke
+`is_trivial(ac)`. Similarly, to find out if `ac` is irreducible,
+one can invoke `is_irreducible`.
+
+Equality of algebraic cycles `ac1` and `ac2` is
+implemented by `ac1 == ac2`.
```

Comment:
Similar to my remark above:
```suggestion
One can check if an algebraic cycle is trivial or irreducible
via `is_trivial` respectively `is_irreducible`.

Equality of algebraic cycles can be tested via `==`.
```
You may also want to turn the reference to `is_trivial` and `is_irreducible` into `@ref` references, though it doesn't really add much here.

However, what *would* be useful if we could somehow convince Documenter to add these places to the index; i.e. find a way so that searching the manual for `is_trivial` also brings up this place. I don't know if Documenter.jl supports that; if it doesn't, we should perhaps file a feature request (or more likely, upvote any existing feature request)... Perhaps @thofma knows?

Reply:
Cf. https://github.com/oscar-system/Oscar.jl/issues/1471